src/screens/ForgetPasswordScreen/styles.ts

```diff
@@ -1,28 +1,34 @@
 import {StyleSheet} from "react-native";
+import {colors} from "../../../assets/colors/colors";
+import {fontFamily} from "../../../assets/fontFamily/fontFamily";
```

Comment:
Font family is semantically a constant, not an asset. Font family only represents the name of the font, not the actual font so that's a constant not an asset. Same goes for `colors` and `fontSizes`